user/common-tasks/backup-emergency-restore-v4.md

```diff
@@ -158,16 +158,12 @@ Emergency Recovery Instructions
 
  7. Decompress and untar the decrypted `private.img` file.
 
-        [user@restore ~]$ cat vm1/private.img.??? | gzip -d | tar -xv
+        [user@restore ~]$ find vm1 -name 'private.img.*[0-9]' | sort -V | xargs cat | gzip -d | tar -xv
         vm1/private.img
 
     **Note:** If your backup was compressed with a program other than `gzip`,
     you must substitute the correct compression program. This information is
-    contained in `backup-header` (see step 4). For example, if you used `bzip2`,
-    then you should do this:
-
-        [user@restore vm1]$ mv private.img.dec private.img.dec.bz2
-        [user@restore vm1]$ bunzip2 private.img.dec.bz2
```

Comment:
Out of curiosity, why remove this part?

Reply:
That was a leftover from older instructions, where the reassembled but still compressed file was written to disk in an intermediate `private.img.dec` file, and then decompressed in a separate step (after make the decompressor recognize the suffix).

But the v4 instructions don't create this file anymore. The data is decompressed as part of the reassemble+decompress+untar pipeline, where `gzip -d` can just be swapped for `bzip2 -d` etc. and it works.

Reply:
Ok, thanks for the explanation. I'll commit a minor rephrasing to make this more perspicuous for newbs like myself.